docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/matrix/inverse-14.rst

```diff
@@ -25,21 +25,21 @@ where :math:`I` is the n-dimensional identity matrix.
 
 The inverse matrix exists if and only if the input matrix is invertible (satisfies any of the properties of the *Invertible Matrix Theorem*). In that case, the inverse exists, and is unique. However, if the matrix is not invertible, the operation may raise an exception or return an undefined result. The operation may return slightly different results for the same input data on different devices due to parallelism and different data types implementations.
 
-This operation can be used to compute the adjugate matrix instead of the inverse.
+This operation can be used to compute the adjoint matrix instead of the inverse.
 
-The adjugate matrix :math:`adj(A)` of a square matrix :math:`A` is defined as:
+The adjoint matrix (conjugate transpose) :math:`adj(A)` of a square matrix of non-complex numbers :math:`A` is simply defined as:
 
 .. math::
 
-   adj(A) = det(A) \cdot A^{-1}
+   adj(A) = (A^{-1})^{T}
```

Comment:
```suggestion
   adj(A) = A^{T}
```

Reply:
it is valid only for real matrices

Reply:
the operation output with adjoint=True computes `(A^{T})^{-1}`

Reply:
Fixed!